steps/pipeline/pipeline.go

```diff
@@ -93,3 +94,15 @@ var _ = gauge.Step("Verify the latest pipelinerun for <state> state", func(state
 	}
 	pipelines.ValidatePipelineRun(store.Clients(), prname, state, "no", namespace)
 })
+
+var _ = gauge.Step("Validate pipelinerun stored in variable <prname> with task <taskname> logs contains <expectedLogs>", func(prname, taskname, expectedLogs string) {
+	logs := cmd.MustSucceed("oc", "logs", "pod"+"/"+store.GetScenarioData(prname)+"-"+taskname+"-pod", "-n", store.Namespace()).Stdout()
```

Comment:
While this works in this particular test, pod might be named differently, e.g. when pipeline run has a long name then it's truncated. More robust way would be to get pod using labels, something like 
```
oc get pod -l tekton.dev/pipelineRun=caches-python-pipeline-run-rnzxb,tekton.dev/pipelineTask=cache-upload
```

Reply:
Sure, `oc logs` is better